src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -0,0 +1,173 @@
+
+                trustedSignersCmd.HelpOption(XPlatUtility.HelpOption);
+                trustedSignersCmd.Description = Strings.TrustCommandDescription;
+
+                trustedSignersCmd.OnExecute(async () =>
+                {
+                    TrustCommand action;
+
+                    if (!command.Values.Any() || string.IsNullOrEmpty(command.Values[0]))
+                    {
+                        action = TrustCommand.List;
+                    }
+                    else if (!Enum.TryParse(command.Values[0], ignoreCase: true, result: out action))
+                    {
+                        throw new CommandLineArgumentCombinationException(string.Format(CultureInfo.CurrentCulture, Strings.Error_UnknownAction, command.Values[0]));
+                    }
+
+                    string name = null;
+                    if (command.Values.Count() > 1)
+                    {
+                        name = command.Values[1];
+                    }
+
+                    string packagePath = null;
+                    string sourceUrl = null;
+                    string fingerPrint = null;
+                    if (command.Values.Count() > 2)
+                    {
+                        if (action == TrustCommand.Author || action == TrustCommand.Repository)
+                        {
+                            packagePath = command.Values[2];
+                        }
+                        else if (action == TrustCommand.Source)
+                        {
+                            sourceUrl = command.Values[2];
+                        }
+                        else if (action == TrustCommand.Certificate)
+                        {
+                            fingerPrint = command.Values[2];
+                        }
+                    }
+
+                    ISettings settings = ProcessConfigFile(configfile.Value());
+
+                    var trustedSignersArgs = new TrustedSignersArgs()
+                    {
+                        Action = MapTrustEnumAction(action),
+                        PackagePath = packagePath,
+                        Name = name,
+                        ServiceIndex = sourceUrl,
+                        CertificateFingerprint = fingerPrint,
+                        FingerprintAlgorithm = algorithm.Value(),
+                        AllowUntrustedRoot = allowuntrustedrootOption.HasValue(),
+                        Author = action == TrustCommand.Author,
+                        Repository = action == TrustCommand.Repository,
+                        Owners = CommandLineUtility.SplitAndJoinAcrossMultipleValues(owners.Values),
+                        Logger = getLogger()
+                    };
+
+                    // MSbuild doesn't pass verbosity parameter means it's normal verbosity for NuGet.
+                    setLogLevel(XPlatUtility.MSBuildVerbosityToNuGetLogLevel(string.IsNullOrWhiteSpace(verbosity.Value()) ? "N" : verbosity.Value()));
```

Comment:
AFAIK, the default verbosity is `Minimal`
https://github.com/NuGet/NuGet.Client/blob/a03a5a8cdd0cec5981fe2c1da6a683a7fee76622/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs#L42-L43

Reply:
With default verbosity `Minimal` there is no log at all. If we don't do this then we get too quiet problem.
But with `nuget.exe` we do get log by default.

Reply:
Addressed.